selenium/integration/tethering/manage.requests.cdf.spec.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { By } from 'selenium-webdriver';
+import { TETHERING_URL } from '../../support/constants';
+import { buildChromeDriver, dataTestId } from '../../support/utils';
+
+describe('Manage tethering requests from the cdf instance', () => {
+  let driver;
+  
+  beforeAll(() => {
+    driver = buildChromeDriver();
+  });
+
+  const navigateToTetheringPage = async () => {
+    await driver.get(TETHERING_URL);
+    await driver.sleep(1000);
+  }
+
+  it('Should navigate to tethering page', async () => {
+    await navigateToTetheringPage();
+    const url = await driver.getCurrentUrl();
+    expect(url.toString()).toContain('/tethering');
+  })
+
+  it('Should successfully accept a tethering request', async() => {
```

Comment:
It looks like this test case depends on the previous case? Each test case should be runnable without any other case. Logic to load the page should go within the case or in a `beforeEach` block.

Reply:
I agree that each test should be independent. However in the codebase, e.g in pipeline related tests, there are many test cases depending on the precedents...

Reply:
Understood, but let's not add to that debt.